packages/core/src/components/datetime-picker/datetime-picker.scss

```diff
@@ -20,9 +21,10 @@
 }
 
 .done {
```

Comment:
Its not part of the PR maybe we need here refactoring but we should keep the classes inside the :host selector